weave/integrations/groq/groq_test.py

```diff
@@ -462,28 +415,23 @@ def run_conversation(user_prompt: str) -> Union[str, None]:
 
     response = run_conversation("What was the score of the Warriors game?")
 
-    weave_server_respose = client.server.calls_query(
-        tsi.CallsQueryReq(project_id=client._project_id())
-    )
-    assert len(weave_server_respose.calls) == 4
+    calls = list(client.calls())
+    assert len(calls) == 4
 
-    flatened_calls_list = [
-        (op_name_from_ref(c.op_name), d)
-        for (c, d) in flatten_calls(weave_server_respose.calls)
-    ]
-    assert flatened_calls_list == [
-        ("run_conversation", 0),
-        ("groq.chat.completions.create", 1),
-        ("get_game_score", 1),
-        ("groq.chat.completions.create", 1),
+    calls_list = [_get_op_name(c.op_name) for c in calls]
```

Comment:
why did you remove the flatten calls? We are not only asserting order, but also depth. Those assertions are important, but lost here

Reply:
Oh I see, I misunderstood what the index represented.

`client.calls()` is already flat and doesn't have that nesting atm.  Should there be an option for that?